contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -118,6 +143,11 @@
                 <field name="incident_polarization" type="NX_NUMBER" recommended="true" units="NX_ANY"/>
             </group>
             <group type="NXelectronanalyser">
+                <group name="model" type="NXfabrication" recommended="true">
+                    <field name="vendor" recommended="true"/>
+                    <field name="model" recommended="true"/>
+                    <field name="identifier" recommended="true"/>
+                </group>
                 <field name="description"/>
                 <field name="energy_resolution" type="NX_FLOAT" recommended="true" units="NX_ENERGY">
```

Comment:
Add class NXresolution

Reply:
I added a class here https://fairmat-nfdi.github.io/nexus_definitions/mpes-refactor/classes/contributed_definitions/NXresolution.html#nxresolution

This only contains the bare resolution yet. We certainly want to add additional information, i.e. how this resolution was determined etc.